internal/config/api-extractor.json

```diff
@@ -136,7 +136,7 @@
 		 * SUPPORTED TOKENS: <packageName>, <unscopedPackageName>
 		 * DEFAULT VALUE: "<unscopedPackageName>.api.md"
 		 */
-		"reportFileName": "api-report.md",
+		"reportFileName": "api-report",
```

Comment:
should this be api-report.api.md? or is that auto-appended?

Reply:
auto-appended